src/game/models/mod.rs

```diff
@@ -27,6 +29,29 @@ impl Add for Vector {
     }
 }
 
+impl Vector {
+    fn new_rand(x_min: f64, x_max: f64, y_min: f64, y_max: f64) -> Vector {
+        Vector {
+            x: rand::random::<f64>() * (x_max - x_min) + x_min,
+            y: rand::random::<f64>() * (y_max - y_min) + y_min,
+        }
+    }
+    fn angle_to_vector(self, other: Vector) -> f64 {
+        let diff_x = other.x - self.x;
```

Comment:
Can this be made simpler by having single `Vector` named `diff` and using the `Sub` trait we created for `Vector`? That way the relationship between `diff_x` and `diff_y` is not implicit (and also reads a little nicer as `diff.x`).

Related #86